src/configuration/DataLink.tsx

```diff
@@ -0,0 +1,192 @@
+
+  return (
+    <div className={className}>
+      <div className={styles.firstRow}>
+        <InlineField
+          label="Field"
+          htmlFor="elasticsearch-datasource-config-field"
+          labelWidth={12}
+          tooltip={'Can be exact field name or a regex pattern that will match on the field name.'}
+        >
+          <Input
+            type="text"
+            id="elasticsearch-datasource-config-field"
+            value={value.field}
+            onChange={handleChange('field')}
+            width={100}
+          />
+        </InlineField>
+        <Button
+          variant={'destructive'}
+          title="Remove field"
+          icon="times"
+          onClick={(event) => {
+            event.preventDefault();
+            onDelete();
+          }}
+        />
+      </div>
+
+      <InlineFieldRow>
+        <div className={styles.urlField}>
+          <InlineLabel htmlFor="elasticsearch-datasource-internal-link" width={12}>
+            {showInternalLink ? 'Query' : 'URL'}
+          </InlineLabel>
+          <DataLinkInput
+            placeholder={showInternalLink ? '${__value.raw}' : 'http://example.com/${__value.raw}'}
+            value={value.url || ''}
+            onChange={(newValue) =>
+              onChange({
+                ...value,
+                url: newValue,
+              })
+            }
+            suggestions={suggestions}
+          />
+        </div>
+
+        <div className={styles.urlDisplayLabelField}>
+          <InlineField
+            label="URL Label"
+            htmlFor="elasticsearch-datasource-url-label"
+            labelWidth={14}
+            tooltip={'Use to override the button label.'}
+          >
+            <Input
+              type="text"
+              id="elasticsearch-datasource-url-label"
+              value={value.urlDisplayLabel}
+              onChange={handleChange('urlDisplayLabel')}
+            />
+          </InlineField>
+        </div>
+      </InlineFieldRow>
+
+      <div className={styles.row}>
+        <InlineField label="Internal link" labelWidth={12}>
+          <InlineSwitch
+            label="Internal link"
+            value={showInternalLink || false}
+            onChange={() => {
+              if (showInternalLink) {
+                onChange({
+                  ...value,
+                  datasourceUid: undefined,
+                });
+              }
+              setShowInternalLink(!showInternalLink);
+            }}
+          />
+        </InlineField>
+
+        {showInternalLink && (
+          <DataSourcePicker
+            tracing={true}
+            onChange={(ds: DataSourceInstanceSettings) => {
+              onChange({
+                ...value,
+                datasourceUid: ds.uid,
+              });
+            }}
+            current={value.datasourceUid}
+          />
+        )}
+      </div>
+
+      <div className={styles.row}>
+        <InlineField label="Base64" labelWidth={12} title="Base64 traceId">
```

Comment:
Can you put the switch just below the "field" input?
Also "Base64" is not very understandable, we could say "Field encoded in base64"
and add a tooltip to explain that quickwit encodes the traceID in base64 by default and Jaeger uses hex

Reply:
Yes I thought about putting a more longer text but we'll have to expend the size of the label with more columns. I'll pick you a new screenshot so you'll able to tell me if it's better or not.

Reply:
@fmassot Is-it better like that ?

<img width="911" alt="Screenshot 2023-12-13 at 11 53 34" src="https://github.com/quickwit-oss/quickwit-datasource/assets/12884347/6e734808-fc69-45a7-9476-cf5ad1ab4cf6">

<img width="884" alt="Screenshot 2023-12-13 at 11 53 41" src="https://github.com/quickwit-oss/quickwit-datasource/assets/12884347/38cb1a2e-719c-4d6b-b341-7f9a17164fb4">

Reply:
yes!